integrations/oci/sdk/runtime/src/test/java/io/helidon/integrations/oci/sdk/runtime/OciExtensionTest.java

```diff
@@ -114,28 +115,27 @@ void potentialAuthStrategies() {
         assertThat(cfg.potentialAuthStrategies(),
                    contains("config", "config-file"));
 
-        config = createTestConfig(ociAuthConfigStrategies("", ""))
+        // this must fail, as empty string is not an allowed strategy
```

Comment:
ok, fine, but please ensure there is a test that when everything is cleared the `potentialAuthStrategies` remain as before.

Reply:
Empty string is not clearing anything. If you think it does, then the test was very wrong in the first place. Empty string is a valid value.
I will check if I can do what you are asking.

Reply:
The test you ask for already exists - you have a test that sends `null`, so it validates what happens if these options are not configured.